libs/dataset-explorer/src/lib/ChartView/LargeDataView/LargeDatasetExplorerTab.tsx

```diff
@@ -162,13 +162,8 @@ export class LargeDatasetExplorerTab extends React.Component<
             <Stack.Item className={classNames.sidePanel}>
               <SidePanel
                 chartProps={this.state.chartProps}
-                cohorts={this.context.errorCohorts.map(
```

Comment:
Please confirm if cohort change, updates bubble chart, thanks.

Reply:
Yes, it doesn't work. I think we probably need props to work with render and this.context for the callbacks.